telemetry/server.go

```diff
@@ -11,17 +11,25 @@ import (
 	"time"
 
 	"github.com/gorilla/mux"
+	"go.uber.org/zap"
 )
 
 type Server struct {
 	Router *mux.Router
 	DB     *sql.DB
+	logger *zap.Logger
 }
 
-func NewServer(db *sql.DB) *Server {
+func NewServer(db *sql.DB) (*Server, error) {
+	logger, err := zap.NewDevelopment()
```

Comment:
In cmd/server we are using `log`.
What do you think if the logger is instantiated there, and passed as a parameter here?, i.e:
```go
func NewServer(db *sql.DB, logger *zap.Logger) (*Server, error) {
	logger := log.Named('server')
```